src/vmm/src/rpc_interface.rs

```diff
@@ -126,88 +129,71 @@ pub enum VmmAction {
 }
 
 /// Wrapper for all errors associated with VMM actions.
-#[derive(Debug, derive_more::From)]
+#[derive(Debug, thiserror::Error, derive_more::From)]
```

Comment:
You can remove `derive_more::From` as `thiserror::Error` covers its functionality with `#[from]`.

Reply:
same as before, I'd like to but, I can't. I 'd need to change all the variants of the enum to derive `thiserror::Error`

Reply:
Resolving this as duplicate of https://github.com/firecracker-microvm/firecracker/pull/3254#discussion_r1087706349